Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml

```diff
@@ -110,6 +110,7 @@
       - EncryptionKeySecurity
       - EncryptionKeyCommand
       - EncryptionKeyCommon
+      - EncryptionKeyPrison
```

Comment:
This should be in the locker, not in the headset by default. See mysta's binary key

Reply:
Instead, maybe consider adding the prison channel to the security encryption key by default

Reply:
Alright, ill get this changed